#!/usr/bin/python3
def raise_exception_msg(messsage=""):
    raise NameError(message)
